### ISSUE TITLE
Clean up (or disable) new lints.

### ISSUE THREAD

**Issue by pq** [dart-lang/linter#407]
https://github.com/dart-lang/linter/commit/9fa9a20bb0de236c727dd2959fb84ac567ec82c4 flips the bit on a bunch of new lints which flag quite a few new warnings.

These should be vetted first and either fixed or disabled.

**Comment by alexeieleusis** [comment]
I think that what happened is that those were introduced in a different commit that was in the works at the same time, will cleanup some time today.

**Comment by pq** [comment]
Sounds good! 👍

**Comment by alexeieleusis** [comment]
I have a problem, `tool/travis.sh` is linting a file under `test/rules`, even though there is
```yaml
  exclude:
    - test/_data/**
    - test/rules/**
```

Any hint?

**Comment by pq** [comment]
Can you send me to the build output?

**Comment by alexeieleusis** [comment]
I lost it now, but I ended up disabling cascase_invocations which was the triggered rule. I have a rewrite pending for it.

**Comment by pq** [comment]
Ok, cool.  Let me know if there's anything I can do to help!
